unable to build with cmake from source

**Describe the bug**
A clear and concise description of what the bug is.
for some reason it broke after git submodule update
**To Reproduce**
Steps to reproduce the behavior:
1. git clone https://github.com/oxen-io/lokinet
2. cd lokinet
3. mkdir build
4.cd build
5. cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF
6. display  CMake Error at external/CMakeLists.txt:12 (message): Submodule 'external/nlohmann' is not up-to-date.
 7.  git submodule update --init --recursive
 8.  cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF
9. make 
**Screenshots Or Logs**
If applicable, add screenshots or log files to help explain your problem.
before git submodule update 
![image](https://github.com/oxen-io/lokinet/assets/24841306/19970e62-a67c-4ec3-8c0d-9cda65d264fd)
https://pastebin.com/TJ7M4hVr
after git submodule update
https://pastebin.com/5vXBZSku
![image](https://github.com/oxen-io/lokinet/assets/24841306/feae0b84-0556-49dd-87e4-1cab633f6f6c)
https://pastebin.com/R0V42drD
**Device and Operating system (please complete the following information):**
 - OS:  arch linux 6.3.1-zen1
-  Device:  PC
 - Lokinet Version number or Git commit hash: latest
 -

--- a/llarp/util/thread/queue_manager.hpp
+++ b/llarp/util/thread/queue_manager.hpp
@@ -1,5 +1,5 @@
 #pragma once
-
+#include <cstdint>
 #include <algorithm>
 #include <atomic>
 #include <cassert>

